src/buildParams.ts

```diff
@@ -51,8 +50,7 @@ const projectDetailsMap: Record<string, ProjectDetails> = {
     launchTimestamp: 1665212400000,
     projectId: Number(process.env.REACT_APP_PROJECT_ID) || 6,
     bundleAmount: 8000,
-    totalBundles: 10000,
-    bundlePercentage: 0.8,
+    totalTokens: 100000000,
```

Comment:
If you're removing the bundlePercentage, shouldn't totalTokens be 80,000,000?

Reply:
Ignore this. Looks like this is used to calculate the royaltyPercentage. In which case, 100m is correct.